pkg/kubectl/cmd/set/set_env.go

```diff
@@ -54,7 +54,8 @@ var (
 		syntax.
 
 		Possible resources include (case insensitive):
-		` + envResources)
+		` + envResources + `
+		Among resources above, environment variables of pod can only be listed, not updated `)
```

Comment:
I prefer like this, @kubernetes/sig-cli-misc 
```
Possible resources include (case insensitive, only can be listed):
pod (po), replicationcontroller (rc), deployment (deploy), daemonset (ds), job, replicaset (rs)
```

Reply:
@shiywang so what about deployment which can be listed and updated.

Reply:
@shiywang what about,   some of them only can be listed

Reply:
@zjj2wry wdyt ? since you implement this command ?

Reply:
i am ok about this. `pod --> pod (po)`
this command alse support remove, should alse aware user.

Reply:
@zjj2wry  I don't understand you very well, can you make it more clear?

Reply:
sorry for delay... i find have a `f.SuggestedPodTemplateResources` func, and you can point pod only suport list not update special